lib/choicesDropdown.js

```diff
@@ -0,0 +1,732 @@
+import Handsontable from 'handsontable'
+import SingleEvent from './singleEvent'
+
+const { addClass, removeClass } = Handsontable.dom
+const { KEY_CODES } = Handsontable.helper
+
+const normalizeOptions = (keyLabel = 'key', valueLabel = 'value') => (options) => (
+  options.map((item) => ({
+    key: item[keyLabel].toString(),
+    value: item[valueLabel].toString(),
+  }))
+)
+
+const isEmptyString = (s) => !s || !s.replace(/\s|\n/g, '').length
+
+const KEY_ATTRIBUTE_NAME = 'key'
+
+const ACTIVE_CLASS_NAMES = {
+  container: 'multi-select__editor',
+  valuesContainer: 'multi-select__editor-input',
+  searchInput: 'multi-select__editor-input-search',
+  options: 'multi-select__editor-options',
+  optionsItem: 'multi-select__editor-options__item',
+  spinnerSpan: 'multi-select__editor-spinner',
+  valueLabel: 'multi-select__editor-label',
+}
+
+export default class ChoicesDropdown {
+  constructor (textareaNode) {
+    /**
+     * Represents if user can choose
+     * more than one value.
+     *
+     * @type {boolean}
+     * */
+    this._isMultiple = !0
```

Comment:
can't you use `true` here instead of `!0`?
additionally, why the `_` prefix?

Reply:
There's a getter for this property. To avoid setter recursion I'm using another variable to store the value - `_isMultiple`.